templates/ovos.config.j2

```diff
@@ -0,0 +1,270 @@
+{
```

Comment:
I would put only the options that are not the default ones.
You could have a look here: https://github.com/OpenVoiceOS/ovos-config/blob/dev/ovos_config/mycroft.conf

This will simplify the template and reduce the maintenance.

Reply:
mhhh, what if we remove just the deprecated? I started exactly from that file, as you can see I have started from the top :D and I ment to go and add all configurations. Do you think it is a bad idea? The biggest problem I have with Ovos is the huge amount of customization, and we have not yet inserted other devices (I have focused on devkit, next I'll try RPI3 headless and MacOS) nor options like hivemind :(

Reply:
A var file per type of hardware should be good.

Reply:
for the most part I use `/etc/mycroft/mycroft.conf` to configure specific plugins to use.  Most of the rest can prob be just left as defaults.

The [raspbian-ovos /etc/mycroft/mycroft.conf](https://github.com/OpenVoiceOS/raspbian-ovos/blob/rewrite/stage-core/01-ovos-core/files/mycroft.conf), but I think the way I have things now, this can be truncated also.